README.md

```diff
@@ -1,12 +1,4 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the Mei Chi Chang's portfolio and SPS projects.
 
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
```

Comment:
Why did you remove this part?

[NO ACTION REQUIRED] This is a sample comment. A reviewer will add such comments to your code to ask for clarifications or suggest some changes.